parachain/pallets/basic-channel/src/outbound/mod.rs

```diff
@@ -30,14 +30,22 @@ pub use weights::WeightInfo;
 )]
 #[scale_info(skip_type_params(M, N))]
 #[codec(mel_bound())]
-pub struct MessageBundle<M: Get<u32>, N: Get<u32>> {
+pub struct MessageBundle<M: Get<u32>, N: Get<u32>, AccountId> {
```

Comment:
Can we have this order for the type parameters: AccountId, M, N

Reply:
May potentially also need to add the following trait bounds for `AccountId`:

```
pub struct MessageBundle<AccountId, ...> where AccountId: Encode, Decode, CloneNoBound, PartialEqNoBound, RuntimeDebugNoBound, MaxEncodedLen, TypeInfo
```

Reply:
Just needed to add the Debug trait instead of RuntimeDebug